plugins/xpressive/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+INCLUDE(BuildPlugin)
+
+SET(CMAKE_CXX_FLAGS "-Dexprtk_disable_sc_andor -Dexprtk_disable_enhanced_features -Dexprtk_disable_break_continue -Dexprtk_disable_comments -Dexprtk_disable_string_capabilities -Dexprtk_disable_rtl_io_file -Dexprtk_disable_rtl_vecops ${WERROR_FLAGS}")
```

Comment:
Oh... now it's not compiling because exprtk.hpp still throws exception in line 2733
@ArashPartow, should I just make a little fix like:
if (stride > 4)
            {
-               throw std::invalid_argument("token_scanner() - Invalid stride value");
+               stride = 4;
            }
????